doc/dev/release_notes.rst

```diff
@@ -3,4 +3,30 @@ Release notes
 
 This page contains the release notes for Catalyst.
 
-.. mdinclude:: ../changelog.md
+.. mdinclude:: ../releases/changelog-0.8.0.md

```

Comment:
Need to remember to also show `-dev.md` here when we merge this to `main`!

Reply:
I think this is the only point to resolve in order to merge this PR with RC @josh146 do you approve?